eos/b-decays/b-to-l-nu.cc

```diff
@@ -129,8 +140,10 @@ namespace eos
     const std::vector<OptionSpecification>
     Implementation<BToLeptonNeutrino>::options
     {
-        { "l", { "e", "mu", "tau" }, "mu" },
-	    { "q", { "c", "u" }, "c"}
+        Model::option_specification(),
+        { "cp-conjugate", { "true", "false" },  "false" },
+        { "l",            { "e", "mu", "tau" }, "mu"    },
+	    { "q",            { "c", "u" },         "c"     }
```

Comment:
this looks like it should be aligned by shifting 4 spaces to the left

Reply:
This is weird, let me check what happened.